source/Audio.cpp

```diff
@@ -301,6 +359,7 @@ void Audio::PlayMusic(const string &name)
 	// Don't worry about thread safety here, since music will always be started
 	// by the main thread.
 	musicFade = 65536;
+	previousTrack.reset(new Music());
```

Comment:
Do you still need this?
If so gib comment

Reply:
done